doc/source/dev/contributor/quickstart_pip.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
+
+.. _quickstart-pip:
+
+======================================================================
+Development environment quickstart guide using ``pip`` on Ubuntu Linux
+======================================================================
+
+This is a high-level overview of what is needed to set up a development
+environment. This is only one possible way out of many. This guide assumes
+you have a fresh install of Ubuntu Linux 20.04, which only has a ``python3``
+executable. We also assume you have already installed ``git`` and cloned 
+the SciPy repository.
+
+
+Installing the system-level dependencies
+----------------------------------------
+
+First, you will also need the compilers for C, C++ and Fortran:: 
+
+    sudo apt install -y gcc g++ gfortran
+    
+SciPy also requires BLAS and LAPACK libraries. You can install several variants
+(ATLAS, OpenBLAS etc), but here we take the simplest option::
+
+    sudo apt install -y liblapack-dev
+
+
+Installing the python-level dependencies
+----------------------------------------
+
+Start with installing ``pip``::
+
+    sudo apt install -y python3-pip
+
+All further work should proceed in a virtual environment. Popular options include
+the standard library ``venv`` module or a separate 
+``virtualenv`` package. There are muliple third-party tutorials on how to
```

Comment:
typo "muliple," but I wasn't going to ask for another round of revision--can just be cleaned up on next opportunity to touch the file

Reply:
Grrr, am adding a spellchecker to the system setup checklist.